src/conduit/components/forms.cljs

```diff
@@ -21,18 +21,39 @@
              (cstr/join ", " v)
              v))])])
 
+(rum/defc InputFieldContainer [value & children]
+  (apply vector :fieldset.form-group children))
+
+(defn TagInputFieldContainer [data errors key]
+  (fn [value & children]
+    (let [{:keys [tagList]} @data]
+      (apply vector :fieldset.form-group children
+             #{[:.tag-list
+                (map (fn [t]
+                       [:span.tag-default.tag-pill {:key t}
+                        [:i.ion-close-round
+                         {:on-click (fn [e] (swap! data assoc :tagList (remove #(= % t) tagList)))}]
+                        t])
+                     tagList)
+                ]}))))
+
 (rum/defc InputField
-  [{:keys [placeholder type value errors on-blur on-focus on-change]}]
-  [:fieldset.form-group
-   [:input.form-control.form-control-lg
-    {:placeholder placeholder
-     :on-change #(on-change (.. % -target -value))
-     :on-blur on-blur
-     :on-focus on-focus
-     :type (or type :text)
-     :value value}]
-   (when errors
-     (InputErrors errors))])
+  [{:keys [placeholder type value errors on-blur on-focus on-change container events]}]
+  (let [input-container (or container InputFieldContainer)
+        input-events (merge {:on-change #(on-change (.. % -target -value))
+                             :on-blur on-blur
+                             :on-focus on-focus}
+                            events)]
+    (input-container
```

Comment:
if you need different container elements this most likely should be two separate components instead